email-smtp/it/AsynchronousIntegrationTest.java

```diff
@@ -59,10 +59,11 @@ public void sendTenMessagesAsynchronouslyWith1PooledThreadAsASmallPerformanceTes
         private boolean finished = false;
 
         @Override
-        public synchronized void edit(@Nonnull final MimeMessage message) throws Exception {
+        public synchronized MimeMessage edit(@Nonnull final MimeMessage message) throws Exception {
```

Comment:
Do you know why we use the `@Nonnull` annotation here? Normally the default is non null... But it's just a question and we shouldn't change it now 😉

Reply:
Developer's decision. As it's not sunrise framework itself, I guess it's fine not to have it following sunrise's guidelines if that's what they preferred :)